docs/references/configuration.rst

```diff
@@ -298,6 +298,21 @@ db-extra-search-path
 
   Multiple schemas can be added in a comma-separated string, e.g. ``public, extensions``.
 
+.. _db-hoisted-tx-settings:
+
+db-hoisted-tx-settings
+----------------------
+
+  =============== ==================================================================================
+  **Type**        String
+  **Default**     statement_timeout, plan_filter.statement_cost_limit, default_transaction_isolation
+  **Reloadable**  Y
+  **Environment** PGRST_DB_HOISTED_TX_SETTINGS
+  **In-Database** pgrst.db_hoisted_tx_settings
```

Comment:
Here it mentions that it can be set by in-db config, but it's not added to:

https://github.com/PostgREST/postgrest/blob/5d3d09923f405bf53c6384a4487fee9195c3118f/src/PostgREST/Config/Database.hs#L44-L46

And there are no entries in  `db_config.sql`  to test this.

Reply:
Done